scripts/jenkins/ardana/gerrit/build_test_package.py

```diff
@@ -108,9 +108,18 @@ def wait_for_build():
     while 'unknown' in sh.osc('results'):
```

Comment:
add "broken" as a transient state here?

Reply:
I wanted to bail after a number of attempts rather than waiting forever on 'broken' builds.

Reply:
understood but "unknown" should be in the same category - I would suggest to treat them the same way (retry for some time, and then give up as an internal error)